sessionmanager.md

```diff
@@ -0,0 +1,91 @@
+# `SessionManager` for @azure/service-bus v7
+
+## Introduction
+
+Sessions allow a user to logically subdivide a queue or subscription
+into multiple smaller queues. This is useful for customers that would 
+like to allow for many smaller streams of messages within a single 
+queue or subscription.
+
+## Use case
+
+A simple use case would be having a set of documents. Users can "edit"
+documents, and each edit is stored as a message within Service Bus.
+
+```javascript
+{ 
+  sessionId: "the document name"
+  body: {
+    mode: "edit",
+    content: "content of the edit"
+  }  
+}
+```
+
+Multiple edits or transformations could be sent for a document, at any time.
+
+To handle this the user would need to write code similar to this:
+
+```javascript
+
+  while (true) {    
+    // some sort of check to make sure we're not processing
+    // too many sessions at a time. The odds of having more documents
+    // than available processors is _high_.
+    acquireSemaphore();
+
+    const sessionReceiver: SessionReceiver = getNextUnlockedSessionReceiver();
```

Comment:
I'm not quite following where this ( getNextUnlockedSessionReceiver();) comes into play.  Could you clarify what this example is showing/what those functions are?  As always apologies if I missed something obvious.

Reply:
I can fill that in with real code! :)

Reply:
Done - it's just a call to sbclient.getSessionReceiver()